gbpservice/neutron/nsf/configurator/api/root_controller.py

```diff
@@ -0,0 +1,16 @@
+from pecan import expose
+from v1 import controllers
+
+
+class RootController(object):
```

Comment:
Does this file comes as part of pecan installation? If so, what is changed in this file from base version to what is checked in?